### DISCUSSION TITLE
🥳  Script Kit Launch Day 🎉

### DISCUSSION THREAD

**Discussion by johnlindquist** [johnlindquist/kit#655] [Announcements]
# Script Kit is Officially Released! 🎉

Download now from https://scriptkit.com

## Free Script Kit Course on [egghead.io](https://egghead.io)

To help you get started, I put together a short course as a feature tour:

https://egghead.io/courses/script-kit-showcase-for-optimizing-your-everyday-workflows-e20ceab4

If you've been using Script Kit for a while on the beta, you know it can do much, much more than what's shown in the lessons, but everyone has to start somewhere. Speaking of the beta...

## Beta Channel Discontinued

If you installed the beta, please download from https://scriptkit.com, quit Kit.app, and replace with the new version. This will put you on the “Main” channel. Updates will be ~monthly. The beta channel is discontinued ❗️

Also, thank you so, so much for all your feedback and patience with updates over the past year. You’ve really helped make Script Kit what it is today and I’m forever grateful 🙏

## Windows Developer Preview

The details for the Windows build are found here: https://github.com/johnlindquist/kit/discussions/654

## Plans for 2022

1. Make the dev setup more contribution-friendly. I would love to accept PRs later next year.
2. Get the Windows build to parity with Mac.
3. Lots of lessons and scripts. I can finally spend more time sharing scripts than working on the app 😎
4. Research into Rust, runtimes, and utilities that can provide any benefit to making our scripts better.
5. Focus on "export to serverless function", "export as github action", and other ways to maximize the work you put into your scripts.
5. Script Kit Pro. A paid version with additional features not found in the base version. Not ready to talk about it, but it's exciting!

**Comment by blackhaj** [comment]
Wow - export to serverless functions and github actions sounds awesome

**Comment by JoepKockelkorn** [comment]
@johnlindquist Will my scripts be wiped when I install the main channel?

**Reply by johnlindquist** [reply]
Nope, Kit.app checks to see if ~/.kenv exists and extracts a bundled tar if not.

Otherwise ~/.kenv stays untouched.

I do recommend keeping scripts under version control, or at least use the "share" menu to create gists for backups just as a general good practice.

**Reply by JoepKockelkorn** [reply]
Thanks for your swift and thorough reply
